portfolio/pom.xml

```diff
@@ -23,7 +23,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
```

Comment:
This seems like the same change that was in PR8 (and some other changes seem to overlap), so I think either that this branch/pr should be based on that one instead of master, or you should only push one or the other.